docs/reference/aggregations/bucket/datehistogram-aggregation.asciidoc

```diff
@@ -10,102 +10,122 @@ that here the interval can be specified using date/time expressions. Time-based
-clocks and calendars. We've invented leap years and leap seconds, standard and
-daylight savings times, and timezone offsets of 30 or 45 minutes rather than a 
-full hour. While these creations help keep us in sync with the cosmos and our
-environment, they can make specifying time intervals accurately a real challenge.
-The only universal truth our researchers have yet to disprove is that a
-millisecond is always the same duration, and a second is always 1000 milliseconds.
-Beyond that, things get complicated.
-
-Generally speaking, when you specify a single time unit, such as 1 hour or 1 day, you
-are working with a _calendar interval_, but multiples, such as 6 hours or 3 days, are
-_fixed-length intervals_.
-
-For example, a specification of 1 day (1d) from now is a calendar interval that
-means "at
-this exact time tomorrow" no matter the length of the day. A change to or from 
-daylight savings time that results in a 23 or 25 hour day is compensated for and the
-specification of "this exact time tomorrow" is maintained. But if you specify 2 or
-more days, each day must be of the same fixed duration (24 hours). In this case, if
-the specified interval includes the change to or from daylight savings time, the
-interval will end an hour sooner or later than you expect.
-
-There are similar differences to consider when you specify single versus multiple
-minutes or hours. Multiple time periods longer than a day are not supported.
-
-Here are the valid time specifications and their meanings:
+==== Calendar and Fixed intervals
 
-milliseconds (ms) ::
-Fixed length interval; supports multiples.
+When configuring a date histogram aggregation, the interval can be specified
+in two manners: calendar-aware time intervals, and fixed time intervals.
 
-seconds (s) ::
-1000 milliseconds; fixed length interval (except for the last second of a
-minute that contains a leap-second, which is 2000ms long); supports multiples.
+Calendar-aware intervals understand that daylight savings changes the length
+of specific days, months have different amounts of days, and leap seconds can
+be tacked onto a particular year.
 
-minutes (m) ::
+Fixed intervals are, by contrast, always multiples of SI units and do not change
+based on calendaring context.
+
+Historically these were configured in a single `interval` field, which lead to
+confusing semantics.  Specifying `1d` would be assumed as a calendar-aware time,
+whereas `2d` would be interpreted as fixed time.  To get "one day" of fixed time,
+the user would need to specify `24h`.
+
+This behavior has been deprecated in favor of two new, explicit fields: `calendar_interval`
+and `fixed_interval`.
+
+===== Calendar Intervals
+
+Calendar-aware intervals are configured with the `calendar_interval` parameter.
+Calendar intervals can only be specified in "singular" quantities of the unit
+(`1d`, `1M`, etc).  Multiples, such as `2d`, are not supported and will throw an exception.
```

Comment:
should we provide more examples hitting this home? I imagine users will hit this exception very quickly. Might be nice to bring them peace by showing this exception

Reply:
Good point.  I can add an example of fixed intervals choking on units like `month` too, just to cover that side as well.